chart/values.yaml

```diff
@@ -1659,6 +1659,8 @@ pgbouncer:
       periodSeconds: 10
       timeoutSeconds: 1
 
+  podAnnotations: {}
```

Comment:
Can you move this up under the deployment annotations?

Reply:
Done

Reply:
Done